init.sh

```diff
@@ -39,6 +42,23 @@ echo "------"
 echo resolver $(awk 'BEGIN{ORS=" "} /nameserver/{print $2}' /etc/resolv.conf | sed "s/ $/;/g") > /etc/api-gateway/conf.d/includes/resolvers.conf
 echo "   ...  with dns $(cat /etc/api-gateway/conf.d/includes/resolvers.conf)"
 
+sync_cmd="echo checking for changes ..."
```

Comment:
So this is assigned to an `echo` command simply to cover cases when the GW runs without syncing its configuration from a remote location. But there may be other services running in the background that may alter the configs ( i.e. the simple bash script doing service discovery using Marathon's API ). So I was thinking to be transparent about what's happening in the background and hence the `echo` cmd. The output should be printed though only when running the container with the `DEBUG` env var set to `true` ( this is still WIP )